lib/wikitrivia/game.ex

```diff
@@ -40,6 +59,41 @@ defmodule Wikitrivia.Game do
     end
   end
 
+  defp start_question(game_id, game_timer_callback) do
+    Agent.update(agent_name_by_game_id(game_id), start_question_phase())
+    game_timer_callback.()
+    Task.async(fn -> execute_after_delay(@question_period_ms, fn -> stop_question(game_id, game_timer_callback) end) end)
+  end
+
+  defp start_question_phase do
+    fn (state) -> %{state | timer_state: :question, timer_data: %{question: "data"}} end
```

Comment:
Question data (or a question id to look up question data with) would be provided here, so that the client doesn't have it until the question state starts.